docker-for-aws/upgrade.md

```diff
@@ -36,6 +36,8 @@ Single containers started (for example) with `docker run -d` are
 **not** preserved during an upgrade. This is because they're not Docker Swarm
 objects, but are known only to the individual Docker engines.
 
+_NIB: Please note that if EFS encryption is enabled, upgrade will [recreate the EFS volume](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html){: target="_blank" class="_"}_
```

Comment:
it's not correct. It WILL RECREATE the volumes no matter what. See details https://github.com/docker/for-aws/issues/138

Reply:
@netflash thanks for the clarification - I missed this part of the comment: `Not having an "Encrypted" property or having it set to false makes a big difference`
Should we simply remove encryption to prevent this odd behavior?

Reply:
Removing that would be the very simple solution.
If you still want to keep it for "Encrypted = true", you may want to deal with the template the way I suggested in the original ticket.

Reply:
I'll update the docs to reflect this change, and the next release will remove encryption as well